mcs/class/System/Test/System.Net.Security/SslStreamTest.cs

```diff
@@ -93,6 +171,167 @@ void AuthenticateClientAndServer (bool server, bool client)
 		}
 	}
 
+	[Category("NotWorking")]    // Works in .Net, doesn't work in mono.
+	[Test()]
+	public void SelfCompatibilityTest()
+	{
+		#if __MonoCS__
```

Comment:
I am sorry, I didn't know ```make run-test-ondotnet``` was a thing.  I will now assume Mono.Security is available on the system that builds the code, but may or may not be available on the system that runs the code.  I'll use a runtime check as Marek originally suggested.  Will push & rebase & squash coming soon.